src/Views/SourceSelector.vala

```diff
@@ -69,11 +69,8 @@ public class OnlineAccounts.SourceSelector : Gtk.Grid {
         add (scroll);
         add (toolbar);
 
-        list_box.row_activated.connect ((row) => {
-            account_selected (((AccountRow) row).account);
-        });
-
         list_box.row_selected.connect ((row) => {
+            account_selected (((AccountRow) row).account);
             remove_button.sensitive = row != null;
```

Comment:
```suggestion
            remove_button.sensitive = true;
```
as you changed the selection mode

Reply:
We do want to set the remove button to insensitive when the listbox is empty though. This would just make the remove button always sensitive since nothing else makes it insensitive

Reply:
alright, true